index.html

```diff
@@ -0,0 +1,156 @@
+        </ul>
+      </div>
+    </div>
+  </header>
+  
+<main>
+  <section class="section1">
+  <h1 class="clipped-text"><span class="h1span">"Take care of your Beauty!"</span> <br> Welcome to Sultan-Oils Company</h1>
+  <p class="paraSection1">Discover the wonders of prickly pear seeds oil at our leading company. Experience the luxurious benefits of our premium skincare and wellness products enriched with essential fatty acids, antioxidants, and vitamins. We prioritize sustainability and ethical sourcing, making a positive impact on both your skin and the environment..</p>
+
+  <div class="event">25 July 2023</div>
+  <div >@Prickly Pear Seeds Oil Beauty Showcase in Tunisia: <br> the Essence of Natural Beauty</div>
+</section>
+<section class="section2">
+  
+  <header>Main Activities</header>
+  <div class="line"></div>
+  <div class="programContainer">
+
+    <div class="prog c1">
+      <img src="images/workshop.png" alt="" class="icon">
+      <h2>Workshops and Demonstrations</h2>
+     <a href="#seemore">SEE MORE</a>
+    </div>
+
+    <div class="prog c2">
+      <img src="images/spa.png" alt="" class="icon">
+      <h2>Spa and Relaxation</h2>
+     <a href="#seemore">SEE MORE</a>
+    </div>
+
+    <div class="prog c3">
+      <img src="images/wellness.png" alt="" class="icon">
+      <h2>Wellness Activities</h2>
+     <a href="#seemore">SEE MORE</a>
+    </div>
+
+    <div class="prog c4">
+      <img src="images/showcase.png" alt="" class="icon">
+      <h2>Product Showcase</h2>
+     <a href="#seemore">SEE MORE</a>
+    </div>
+
+    <div class="prog c5">
+      <img src="images/pro1.png" alt="" class="icon">
+      <h2>Expert Panel and Q&A Sessions</h2>
+     <a href="#seemore">SEE MORE</a>
+    </div>
+  </div>
+  <div class="joinUs "><a href="">Reserve Your Spot Now</a> </div>
+  <div class="lecture"></div>
+</section>
+
+<section class="section3 Feature-speakers">
+  <h2>Founders</h2>
+  <div class="line"></div>
+  <div class="speakers">
+
+<div class="person ">
+    <img class="personImage"  src="images/peter2.jpg"alt="peter_image">
+  <div class="clm2">
+    <h2 class="name">Peter</h2>
+     <p class="p1">Peter, our accomplished CEO, leads with vision and expertise, propelling our company to new heights of success. His strategic acumen and commitment to innovation drive our team to deliver exceptional results.</p>
+     <div class="line2"></div>
+     <p class="p2">Chief Executive Officer (CEO)</p>
+  </div>
+</div>
+
+<div class="person">
+  <img class="personImage"  src="images/mayssa5.webp"alt="Mohamme_image">
+<div class="clm2">
+  <h2 class="name">Mayssa</h2>
+   <p class="p1"> Responsible for developing and executing the company's marketing strategies. </p>
+   <div class="line2"></div>
+   <p class="p2">Chief Marketing Officer (CMO)</p>
+</div>
+</div>
+<div id="more" class="more">MORE
+  <img src="images/arrow_down 1.png" alt="arrow-icon">
+</div>
+
+</div>
+
+</section>
```

Comment:
- [ Required ] I think it would be better if we do as requested by the previous reviewer and ensure that this section is generated dynamically using JavaScript. You've already done this in `main.js` so just remove this markup from your page and ensure that the speakers list is generated dynamically.